packages/styles/src/scripts/update-test-images.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
If I understand correctly, will the script download the report if we provided the url? If this is the case, could you add on the readme how to execute the script and a second part as an alternative how to do it in manual?